demos/transformer/README.md

```diff
@@ -62,5 +62,5 @@ manual steps.
     app and select a MediaPipe-based effect.
 
 [Transformer]: https://developer.android.com/media/media3/transformer
-[MediaPipe]: https://google.github.io/mediapipe/
-[build an AAR]: https://google.github.io/mediapipe/getting_started/android_archive_library.html
+[MediaPipe]: https://developers.google.com/mediapipe
```

Comment:
Can we update this to https://ai.google.dev/edge/mediapipe/solutions/guide?

This link is already redirecting there